src/site/content/en/blog/css-individual-transform-properties/index.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: Finer grained control over CSS transforms with individual transform properties
+authors:
+  - bramus
+  - dbaron
+description: >
+  Learn how you can use the individual translate, rotate, and scale CSS properties to approach transforms in an intuitive way.
+date: 2022-08-02
+---
+
```

Comment:
Add the aside as in this post https://web.dev/transformstream/

I'll go through and tidy these up once we're not "newly interoperable" any more, as I can find them via the newly-interoperable tag.